students/ravi/Session02/fizz_buzz.py

```diff
@@ -0,0 +1,11 @@
+for i in range(100):
```

Comment:
You can add +1 to the range function here to make the list of numbers provided include 100. Print out range(100) and print out range(100+1) to see the difference.